server/src/test/java/io/spine/server/entity/given/EntityTestEnv.java

```diff
@@ -100,4 +104,34 @@ public EntityWithMessageId() {
             super(Sample.messageOfType(ProjectId.class));
         }
     }
+
+    public static class TestAggregate extends Aggregate<String, StringValue, StringValueVBuilder> {
```

Comment:
Please create an issue saying that currently we use aggregates in tests for `Entity`, but it seems odd. There should be a cleaner way of doing it. And put the reference to the issue as a todo item here.